functions/index.js

```diff
@@ -0,0 +1,153 @@
+const axios = require('axios');
```

Comment:
can we use es5 syntax and typescirpt? like import { a, b } from 'package'. can address later but we should consider being consitent across this repo and the other

Reply:
will do this as a separate issue